README.md

```diff
@@ -8,32 +11,94 @@
 [![SpigotMain - ImprovedFactions](https://img.shields.io/badge/Spigot-ImprovedFactions-orange)](https://www.spigotmc.org/resources/improved-factions.95617 "Spigot ImprovedFactions page")
 [![CodeFactor](https://www.codefactor.io/repository/github/toberocat/improvedfactions/badge)](https://www.codefactor.io/repository/github/toberocat/improvedfactions)
-<a href="https://www.paypal.com/donate/?hosted_button_id=BGB6QWR886Q6Y">
-    <img src="https://img.shields.io/badge/Donate-PayPal-green.svg?logo=paypal&style=flat-square" height="20" alt="Donate">
-</a>
-<a href="https://discord.gg/VmSbFNZejz">
-    <img src="https://img.shields.io/discord/819932760550014986.svg?logo=discord&colorB=7289DA&style=flat-square" height="20" alt="Discord chat">
-</a>
+[![Donate - PayPal](https://img.shields.io/badge/Donate-PayPal-green.svg?logo=paypal&style=flat-square)](https://www.paypal.com/donate/?hosted_button_id=BGB6QWR886Q6Y)
+[![Discord](https://img.shields.io/discord/819932760550014986.svg?logo=discord&colorB=7289DA&style=flat-square)](https://discord.gg/VmSbFNZejz)
+
 </div>
 
-## Documentation
+## Table of Contents
+- [About](#about)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Configuration](#configuration)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [Support](#support)
+- [License](#license)
 
-<div align="center">
+## About
+
+ImprovedFactions is a lightweight and high-performance Minecraft faction plugin. It offers a user-friendly experience with features that can be downloaded directly from Minecraft, enhancing server gameplay and community interaction.
+
+## Features
+
+- **Performance Optimized**: Designed for efficiency, ensuring smooth gameplay even on busy servers.
+- **User-Friendly**: Intuitive commands and interfaces for players of all experience levels.
+- **Customizable**: Extensive configuration options to tailor the faction experience to your server's needs.
+- **Minecraft Integration**: Unique features downloadable directly through Minecraft for seamless user experience.
+- **Regular Updates**: Continual improvements and new features to keep your faction gameplay fresh and exciting.
+
+## Installation
+
+1. Download the latest version of ImprovedFactions from the [Spigot page](https://www.spigotmc.org/resources/improved-factions.95617) or [GitHub releases](https://github.com/ToberoCat/ImprovedFactions_new/releases/).
+2. Place the downloaded `.jar` file in your server's `plugins` folder.
+3. Restart your server.
+4. The plugin will generate default configuration files, which you can then customize.
+
+## Usage
+
+- `/f help` - View all available faction commands
+- `/f create <name>` - Create a new faction
+- `/f join <faction>` - Join an existing faction
+- `/f claim` - Claim land for your faction
+
+For a full list of commands and their usage, please refer to our [documentation](#documentation).
```

Comment:
Change redirect to https://github.com/ToberoCat/ImprovedFactions_new/wiki instead of #documentation,
much less clicks to do :)

Reply:
This should really go to https://github.com/ToberoCat/ImprovedFactions_new/tree/main/docs
since that all the wiki part is for

Reply:
> This should really go to https://github.com/ToberoCat/ImprovedFactions_new/tree/main/docs since that all the wiki part is for

Alright. Lets do it